Focused on Localization for Taiwanese People and fixed many mistranslations.

HJ-zhtw opened the pull request:
Focused on Localization for Taiwanese People and fixed many mistranslations.
Basic on the original TC translations, this change can help Taiwanese people better know what the strings mean.
What have I changed? e.g. Still in Simplified: 后→後, 鬆鼠→松鼠, 虫→蟲 ... and many other., Localization not for Taiwan(zhTW): 兼容→相容, 性能→效能, 激活→啟動, and so on. Integration for Naga the boss: 娜迦、納迦→那伽 (According to WikiPedia.)
Sum it up, I only changed it for localization, not for new-added strings.

Benimatic commented:
I see you've sent this also to https://github.com/TeamTwilight/twilightforest which is the correct repository.

HJ-zhtw commented:
@Benimatic Yes, I missent it. Sorry for disturbing.

3TUSK committed:
Update zh_cn.lang

3TUSK committed:
Update zh_cn.lang

Andromander committed:
Rename Highlands Center to Final Plateau

bs2609 committed:
Fix #658. Thanks implicit fallthrough!

bs2609 committed:
Clean up a couple of switch blocks

AtomicBlom committed:
Add support for creating maven artifacts

Drullkus committed:
Dependency update

Drullkus committed:
Add moar Emissive stuff

Drullkus committed:
Groundwork for Magic Tree related Bee species for Forestry

bs2609 committed:
Some code cleanup for the magic trees

bs2609 committed:
More magic tree tweaks

bs2609 committed:
Update trophy pedestal to use status bar messages

bs2609 committed:
Add option for extra portal creation checks, slightly WIP still

bs2609 committed:
Some portal code fixes and cleanup

bs2609 committed:
More tweaking

bs2609 committed:
Finish tidying

bs2609 committed:
Reorganise a bit of teleporter code

bs2609 committed:
Only register ruins with valid weights

bs2609 committed:
More ruin list fixes

bs2609 committed:
Fix broken advancement reference

bs2609 committed:
Improve custom spawn type logic

noobanidus committed:
Restrict "Twilight Hunter" achievement to the Twilight Forest dimension.

This way, if mobs are spawned outside of the Twilight Forest (for
whatever reason), there isn't a weird circumstance where the player
achieves "Twilight Hunter" before even entering a portal for the first
time.

noobanidus committed:
Move raven feathers into the OreDictionary and remove extraneous arrow recipe. (#671)

Andromander committed:
Fix another broken advancement reference (#674)

Andromander committed:
Tower Devices

Initial progress

Drullkus committed:
Commit some other refactors after ragequitting the Druid Hut refactor

Drullkus committed:
Cleanup of 4 am programming: Why did I put suppliers?

Drullkus committed:
Custom Bighorn Loot Tables

They just defer to vanilla sheep variants

Drullkus committed:
Add Unbreaking Glass Swords to JEI/Creative, changes to Patchouli as well

Drullkus committed:
New Item Usage Trigger which will activate when certain items are used on certain blocks.

Only Magic Bean currently use this because I'm tired and sick af so I'll add this trigger to other TF stuffs.

Drullkus committed:
Localizations are why we can't have nice things

Drullkus committed:
Patchouli pages on Phantom Armor

Drullkus committed:
Edit Charms' entry, and add page stuffs for Crumble Horn

bs2609 committed:
Add support for colour mixing to bighorn sheep

Drullkus committed:
Musical Beans - and other Patchouli edits

Drullkus committed:
Terrorcotta!

Drullkus committed:
A silly, nonsensual, and totally awesome pun

Drullkus committed:
Add recipe condition to disable Uncrafting Table crafting if disabled

Drullkus committed:
Quick edit for Death Tome's hurt loottable

Andromander committed:
Pages for fire_jet blocks

Includea Smoking Block, Encased Smoker, Fire Jet, and Encased Fire Jet.
Sortnum reference file updated as such

Andromander committed:
Wrong advancements

Andromander committed:
Do I fix this absolutely tiny typo? Answer is yes

bs2609 committed:
Simple uncrafting condition that doesn't make people yell at you

bs2609 committed:
Always compile using UTF-8

bs2609 committed:
Tidier formatting

Drullkus committed:
Version :b:ump

Drullkus committed:
Cut some cruft from naga entry, for now, until we get proper content

bs2609 committed:
Fix CME in armour trigger and tidy up code a bit

bs2609 committed:
Fix indentation

bs2609 committed:
Fix all structures losing conquer status

bs2609 committed:
More advancement trigger fixes

williewillus committed:
Big cleanup in lich renderer, properly separate lich model from shield rendering

williewillus committed:
Actually add new file

williewillus committed:
Minor DRY in models

bs2609 committed:
Stop Tree of Transformation slowing down too much due to randomness

bs2609 committed:
Small code cleanup

bs2609 committed:
Update some variable names to more current mappings

bs2609 committed:
Add missing properties to wooden (trap)doors

bs2609 committed:
Add map colours to new wood blocks

Mojang please make your classes properly extensible

bs2609 committed:
Add correct item drops to wooden doors

bs2609 committed:
Add custom map colours for some compressed blocks, plus tidying

bs2609 committed:
Bump Forge version and clean up game rule handling

bs2609 committed:
Make ravens also count towards the 'twilight_hunter' advancement

bs2609 committed:
Fix magic leaves inheriting vanilla sapling drops, resolves #698

bs2609 committed:
Fix up some map colours, closes #700

bs2609 committed:
Some code tidying

bs2609 committed:
Some more map colour work

bs2609 committed:
Some fixes for troll roots

bs2609 committed:
Fix the ore map checks to work like the ore magnet

bs2609 committed:
Tidy up some stray copypasta

bs2609 committed:
Couple more map colour tweaks

bs2609 committed:
Fix minor porting mistake in some block drop code

bs2609 committed:
Small cleanup of some other block drop code

bs2609 committed:
More code updates

bs2609 committed:
Move some structure-based advancements to being data-driven

bs2609 committed:
Adjust hollow hill structure box to better fit hill size

bs2609 committed:
Bit more tweaking

noobanidus committed:
Replace umlaut-o with plain o. Resolves build/compilation issues. (#692)

bs2609 committed:
Clean up a few translation keys

bs2609 committed:
Fix some bad item model handling

bs2609 committed:
Merge branch '1.12.x' of https://github.com/TeamTwilight/twilightforest into 1.12.x

bs2609 committed:
Add other boss spawner variants to the creative menu

bs2609 committed:
Clean up some model registration code

Drullkus committed:
1.13 future thing

Drullkus committed:
Initial commit for Twisted Stone Pillars, still WIP

bs2609 committed:
Fix FOV-changing logic for bows

bs2609 committed:
Add forwards-compatibility handling to FOV changes

bs2609 committed:
Improve protection box handling a bit

bs2609 committed:
Small harvest check code cleanup

bs2609 committed:
Clean up a bunch of model/rendering code

bs2609 committed:
Add some missing WorldProvider overrides, fixes #713

bs2609 committed:
Apply magic map fixes to maze map

Drullkus committed:
Far too much time blown on these dang pillars

Drullkus committed:
Some clean-up, because by the good lord, I made a mess

bs2609 committed:
Update item colour handler list to match item registration

bs2609 committed:
Improve some structure generator handling

bs2609 committed:
Fix some unresolved texture errors

bs2609 committed:
Some small GL state cleanups

bs2609 committed:
Add missing class check to upper goblin knight, fixes #720

bs2609 committed:
Fix the other data fixer class name

bs2609 committed:
Clean up a lot of ResourceLocation handling

bs2609 committed:
More structure generator cleanups

bs2609 committed:
Couple of random fixes

bs2609 committed:
Use HTTPS for Maven URLs where applicable

bs2609 committed:
Clean up some particle handling

bs2609 committed:
Add some missing isRemote checks

bs2609 committed:
Clean up handling for protected entities, spawn particles properly

bs2609 committed:
Code cleanup and minor improvements for boss spawners

bs2609 committed:
Clean up another block type cast

bs2609 committed:
Quick size optimisation pass over new textures

bs2609 committed:
Add missing lang file entries for world-gen settings

bs2609 committed:
Tidy up a couple of translation keys

bs2609 committed:
Don't forget this file

bs2609 committed:
Fix some tree feature generators being broken

bs2609 committed:
Clean up a bunch of biome code

bs2609 committed:
Fix some block flags used in world generation

bs2609 committed:
Some fixes for vine generator, closes #732

bs2609 committed:
Add y-level check to some world-gen spawning code

bs2609 committed:
Some extra checks for existing portals

bs2609 committed:
Fix particles spawned via item interactions

bs2609 committed:
Tidy up unused imports

bs2609 committed:
Nullability fixes and some small tidying

bs2609 committed:
Some code de-duplication

bs2609 committed:
Some more code cleanup

Tamaized committed:
fix isPositionOccupied and reduce a little bit of dupe code

Tamaized committed:
am big dum

Neerwan committed:
fr_fr.lang (#734)

bs2609 committed:
Bump Forge version, fix #602

bs2609 committed:
Couple of quick fixes

Tamaized committed:
jredfox is stupid, closes #737

it's a feature

bs2609 committed:
Some code cleanups and renaming for weather renderer

bs2609 committed:
Quick sweep over sky renderer too

bs2609 committed:
Improve weather effect update logic

bs2609 committed:
Spinny sliders

bs2609 committed:
More renames

bs2609 committed:
Another small cleanup pass

bs2609 committed:
Tidy up logging statements

bs2609 committed:
More ResourceLocation cleanup

bs2609 committed:
Some repackaging

bs2609 committed:
Fix a missed reference

bs2609 committed:
illegal deviance

bs2609 committed:
More cleanups

Tamaized committed:
added 5 more music tracks, closes #405

Thanks MrCompost!!

also removed vanilla music from TF

Tamaized committed:
new alpha yeti sounds #269

Tamaized committed:
added higher pitch throw sounds to the yetis too

Tamaized committed:
alpha yeti will now stop shooting ice bombs if the player is too close

Hubry committed:
IMC and config support for hollow hill stalactites (#729)

bs2609 committed:
Small post-merge cleanup

bs2609 committed:
Tidy up some giant block logic and a couple of other things

Tamaized committed:
add deer sounds #269 closes #362

they still moo kind of

Tamaized committed:
give yeti's higher pitch alpha yeti idle/hurt/death sounds #269

Tamaized committed:
add wolf sounds #269

Tamaized committed:
add death tome sounds #269

Tamaized committed:
Merge branch '1.12.x' of https://github.com/TeamTwilight/twilightforest into 1.12.x

Tamaized committed:
anotha one

Tamaized committed:
FULL BLAST

bs2609 committed:
Generify some AI tasks

bs2609 committed:
Some material tweaks

bs2609 committed:
Minor tidying of chisel support

bs2609 committed:
More random block variants for minotaur maze walls

bs2609 committed:
Small improvements to cave generators

bs2609 committed:
Chunk primer optimisations

bs2609 committed:
Move some client classes to client package

bs2609 committed:
Another small uncrafting cleanup

bs2609 committed:
Bump Forge version, fix issues with items that take damage during use

bs2609 committed:
Clean up bow code

bs2609 committed:
Couple of config handling fixes

bs2609 committed:
Small code tidy for advancement checks

bs2609 committed:
Small logic fix and tidying for portal formation code

bs2609 committed:
Also clean up portal logic in tick handler

bs2609 committed:
Tidy up progression rule checks a bit

bs2609 committed:
Another event handler tidy-up

bs2609 committed:
Small improvements to seeker arrows

bs2609 committed:
Proper parrot sound events

bs2609 committed:
Fix wrong effect ID for hydra mortars, fixes #752

bs2609 committed:
Add custom biome type to twilight forest biomes

bs2609 committed:
Clean up a bunch of chunk generator checks

bs2609 committed:
IntelliJ please remember my selected changes :<

bs2609 committed:
Mark labyrinth as completed on Minoshroom kill

Drullkus committed:
Fully address #520, rename a few params and methods properly

Drullkus committed:
Merge branch '1.12.x' of https://github.com/TeamTwilight/twilightforest into 1.12.x

bs2609 committed:
Small ray-trace logic improvements

Drullkus committed:
Not sure what this clone() is doing here

bs2609 committed:
Improve critter placement logic, should resolve #759

bs2609 committed:
Make animated item renderers a bit safer

bs2609 committed:
Clean up animated stack rendering code too

bs2609 committed:
Make Fire Beetle immune to fire, closes #765

bs2609 committed:
Add missing capability check

bs2609 committed:
Clean up this item handling code too

bs2609 committed:
Update bauble handling code to better match main inventory equivalent

baguchi committed:
add naga eye close when naga Dazed (#762)

noobanidus committed:
Additional loot tables as per #741 (#742)

closes #741

bs2609 committed:
Small statemapper cleanup

bs2609 committed:
Fix a few mismatches with vanilla code for triple bow

bs2609 committed:
Small pathfinding update

bs2609 committed:
More texture optimisations

bs2609 committed:
More AI task updating

baguchi committed:
One Phantom Knight spawns with a shield on Hard (#782)

baguchi committed:
add guard cooldown and add blocking sound (#783)

klmendor committed:
Modified ru_ru.lang. Corrected translation (#785)

jellysquid3 committed:
Improve terrain generation times slightly (#787)

jellysquid3 committed:
Reduce the amount of CPU time spent in TFFeature (#786)

bs2609 committed:
Fix biome block replacement

bs2609 committed:
Fix incomplete feature generation

bs2609 committed:
Feature lookup fixes

Tamaized committed:
Alpha Yeti now throws ice bombs during a rampage

Tamaized committed:
Deer now flee from players

Tamaized committed:
Version Bump

baguchi committed:
The Snow Queen now handles attacking non-player correctly also Ice Cores now melt, closes #302. (#790)

baguchi committed:
New Minoshroom smash ground attack (#761)

Tamaized committed:
Flesh out the new Minoshroom attack

baguchi committed:
BlockGoblins will now launch their spiky block flails towards a target (#772)

bs2609 committed:
More improvements for dark forest canopy generation

bs2609 committed:
Clean up biome/feature mapping

bs2609 committed:
Fix some leaf placement by tree generators

bs2609 committed:
Tweak shield capability handling (and fix typo)

bs2609 committed:
Bit more shield cleanup

bs2609 committed:
Improve chunk creation speed a bit

bs2609 committed:
Clean up terrain deformation for structures

bs2609 committed:
Bit more cave-gen tidying up

bs2609 committed:
Move some creature-spawning code around

bs2609 committed:
Clean out some old instanceof checks

bs2609 committed:
Some trait adjustment

bs2609 committed:
Bit more shield code tweaking

Tamaized committed:
birbs will try their very best to avoid drowning

Tamaized committed:
Fixed uberous soil render without CTM

bs2609 committed:
Improve death checks for phantom knights

comp500 committed:
Fix some wood door/trapdoor recipes (#812)

bs2609 committed:
Fix another door recipe

bs2609 committed:
Add advancement conditions

Tamaized committed:
guys i added the final boss OH MY GODDDDD

bs2609 committed:
Trust in birbs to float

Tamaized committed:
Urghast tweaks

 - Shorter threshold until a trantrum
 - Traps ensure the threshold resets
 - Take 1/10th damage instead of 1/4th while in a tantrum
 - Don't take any knockback

Andromander committed:
More pages, swamp theme

Tamaized committed:
Spooky Forest WIP

Andromander committed:
Requested edits

Tamaized committed:
finalized spoop

Tamaized committed:
OOPSIES

Alepod committed:
update ru_ru.lang (#799)

3TUSK committed:
Update zh_cn.lang (#815)

Tamaized committed:
Version Bump

Tamaized committed:
bruh

bs2609 committed:
Pre-change indentation fix

bs2609 committed:
Cleanup advancement trigger checks

Tamaized committed:
bump jei and forge

deps are annoying

Tamaized committed:
absolute bruh moment

Tamaized committed:
hopefully this pom works... :eyes:

Tamaized committed:
haha doesnt work.. worth the shot

https://stackoverflow.com/a/55277588

Drullkus committed:
Quick shortcircuit for Bresenham line

Drullkus committed:
Filter Charms to players only
